lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -313,6 +313,10 @@ class DetailsRenderer {
             }
             break;
           }
+          case 'link': {
+            itemElement = this._renderLink({url: value, text: value});
```

Comment:
Need to add some typescript typings here for stringifying this value.
```javascript
const strValue = /** @type {string} */ (value);
itemElement = this._renderLink({type: 'link', url: strValue, text: strValue});
```

Reply:
Linter recommended me to add a type in the following interface in `lhr-lite.d.ts`:
```
 export interface ColumnHeading {
          /** The property key name within DetailsItem being described. */
          key: string;
          /** Readable text label of the field. */
          label: string;
          // TODO(bckenny): should be just string and let lhr be more specific?
          valueType: 'url' | 'timespanMs' | 'bytes' | 'thumbnail' | 'link';
        }
```
And it no longer gave me an error. Perhaps this could work?